studio/components/grid/store/reducers/row.ts

```diff
@@ -136,14 +136,6 @@ const RowReducer = (state: RowInitialState, action: ROW_ACTIONTYPE) => {
         }),
       }
     }
-    case 'REMOVE_ROWS': {
-      const totalRows = state.totalRows - action.payload.rowIdxs.length
-      return {
-        ...state,
-        rows: state.rows.filter((x) => !action.payload.rowIdxs.includes(x.idx)),
-        totalRows: totalRows,
-      }
-    }
```

Comment:
@joshenlim when `REMOVE_ROWS` is dispatched, `rows` and `totalRows` are already updated.